extensions/rich_text_components/richTextComponentsRequires.ts

```diff
@@ -25,6 +25,9 @@ require(
 require(
   'rich_text_components/Link/directives/' +
   'oppia-noninteractive-link.directive.ts');
+require(
+  'rich_text_components/Concept/directives/' +
+  'oppia-noninteractive-concept.directive.ts');
```

Comment:
Please move this after the Collapsible require to keep the alphabetical ordering.

Reply:
Done